Felienne/hedy
# [FIX] Enables developers to create css file with all Tailwind classes

**Description**
Before updating to Tailwind v3 the `generate-development-css` file automatically generated all possible Tailwind classes. Making front-end development easier as we don't have to manually generate the .css files after adding classes. However, with the update to Tailwind v3 the `purge` option was deprecated as there was no longer an option to generate the full css file.

As pointed out by @rix0rrr the current script doesn't work anymore. In this PR we implement a work-around to still enable us to generate the complete css file. As this is no longer supported it is a bit hacky-tacky as discussed here:
https://github.com/tailwindlabs/tailwindcss/discussions/6557

**Fixes**
This PR fixes #2039

**How to test**
Run the `generate-development-css` script and verify that a css file is generated containing all Tailwind classes. Note that we copied the `tailwind.config.js` file to `tailwind.full.js` as for some reason the container width is messed up when we use the original full config file.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
